app/views/transactions/_form.html.slim

```diff
@@ -1,41 +1,53 @@
+            selected: f.object.category_id
 
-    .col-md-3.col-sm-6.col-xs-6
-      = f.input :customer_name, label: false,
-        input_html: { data: { url: autocomplete_customers_path,
-          placeholder: transaction.find_customer_name_by_id(f.object.customer_id),
-          value: @q.nil? ? transaction.find_customer_name_by_id(f.object.customer_id) \
-            : (@q.customer_id_eq ? transaction.find_customer_name_by_id(@q.customer_id_eq) : nil) } }
+    .row
+      .col-md-6.col-sm-12.col-xs-12
+        = f.input :customer_name, label: false,
+          input_html: { data: { url: autocomplete_customers_path,
+            placeholder: 'Customer',
+            value: @q.nil? ? transaction.find_customer_name_by_id(f.object.customer_id) \
+              : (@q.customer_id_eq ? transaction.find_customer_name_by_id(@q.customer_id_eq) : nil) } }
 
-    .col-md-3.col-sm-6.col-xs-6
-      - if f.object.bank_account.nil? || f.object.bank_account.visible?
-        = f.association :bank_account, as: :grouped_select, label: false,
-          collection: current_organization.bank_accounts.visible.positioned.grouped_by_currency(current_organization.default_currency),
-          prompt: 'Bank account', group_method: :last,
-          selected: @q.nil? ? f.object.bank_account_id \
-            : (@q.bank_account_id_eq && current_organization.bank_accounts.find(@q.bank_account_id_eq).visible? \
-            ? @q.bank_account_id_eq : nil)
-      - else
-        = f.input :bank_account, label: false, disabled: true
-    .col-md-3.col-sm-6.col-xs-6
-      = f.input :comment, label: false
-    .col-md-3.col-sm-6.col-xs-6
-      = f.input :date, as: :string, label: false, placeholder: 'Date',
-        input_html: { class: 'datepicker', value: f.object.date.try(:strftime, '%d/%m/%Y') }
-  .col-sm-3
-    = render 'transactions/btn_block', f: f, transaction: transaction, selected: 'Transaction'
+      .col-md-6.col-sm-12.col-xs-12
+        - if f.object.bank_account.nil? || f.object.bank_account.visible?
+          = f.association :bank_account, as: :grouped_select, label: false,
+            collection: current_organization.bank_accounts.visible.positioned.grouped_by_currency(current_organization.default_currency),
+            prompt: 'Bank account', group_method: :last,
+            selected: @q.nil? ? f.object.bank_account_id \
+              : (@q.bank_account_id_eq && current_organization.bank_accounts.find(@q.bank_account_id_eq).visible? \
+              ? @q.bank_account_id_eq : nil)
+        - else
+          = f.input :bank_account, label: false, disabled: true
+    .row
+      .col-md-6.col-sm-12.col-xs-12
+        = f.input :comment, label: false
+      .col-md-6.col-sm-12.col-xs-12
+        = f.input :date, as: :string, label: false, placeholder: 'Date',
+          input_html: { class: 'datepicker', value: f.object.date.try(:strftime, '%d/%m/%Y') }
+  .col-sm-12
+    = f.button :submit, class: 'btn-primary'
```

Comment:
@yunixon but this buttons in modal footer. Primary button (submit) should be on right side.